src/inmanta/protocol/endpoints.py

```diff
@@ -460,3 +481,7 @@ async def _call(
 
         result = await self._transport_instance.call(method_properties, args, kwargs)
         return result
+
+    def close(self) -> None:
+        if self._transport_instance and self._transport_instance.client:
+            self._transport_instance.client.close()
```

Comment:
The documentation is unclear on the behavior of `close` when the singleton objects are used.